web/src/lib/components/Navbar.svelte

```diff
@@ -46,13 +46,51 @@
         }
     }
 
+    function closeMenu() {
+        menuDisplay = "hidden";
+    }
+
+    function openMenu() {
+        menuDisplay = "block";
+    }
+
+    function toggleMenu() {
```

Comment:
having 3 functions just to toggle a boolean seems overkill...

let's have just a boolean that stores the visibility, then the toggle just becomes on:click={() => menuVisible = !menuVisible} and you can use a Svelte conditional class based on the "visible" boolean.